drupal8-example/docker-compose.yml

```diff
@@ -21,16 +21,56 @@ services:
     working_dir: /var/www/web
     volumes:
       - './:/var/www:rw,cached'
+      - './docker/my_init.d/install-cron.sh:/etc/my_init.d/install-cron.sh'
     environment:
       PATH: '/var/www/vendor/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin'
       PHP_SENDMAIL_PATH: /usr/local/bin/mhsendmail --smtp-addr="mailhog:1025"
+      DRUSH_OPTIONS_URI: https://MY-PROJECT.docker
 
   db:
-    image: reload/db-datadir:reloaddk-latest-fulldb
+    image: mariadb:10
```

Comment:
Sorry, missed a minor. The datadir builder uses mariadb:10.3.9 - we might as well keep them in sync. (https://github.com/reload/docker-db-datadir-builder/blob/master/src/build-datadir-image.sh#L145). I mainly lock the version down as I have to make a tweak to the entrypoints script - when https://github.com/docker-library/mysql/pull/471 eventually gets resolved and makes it into mariadb we can probably slack the version constraint a bit.

Reply:
I pushed a commit locking this one down as well.